pkg/backend/httpstate/backend.go

```diff
@@ -625,7 +625,28 @@ func (b *cloudBackend) RenameStack(ctx context.Context, stackRef backend.StackRe
 		return err
 	}
 
-	return b.client.RenameStack(ctx, stack, string(newName))
+	// Support a qualified stack name, which would also rename the stack's project too.
+	// e.g. if you want to change the project name on the Pulumi Console to reflect a
+	// new value in Pulumi.yaml.
+	newRef, err := b.ParseStackReference(string(newName))
+	if err != nil {
+		return err
+	}
+	newIdentity, err := b.getCloudStackIdentifier(newRef)
+	if err != nil {
+		return err
+	}
+
+	// Transferring stack ownership is available to Pulumi admins, but isn't quite ready
+	// for general use yet. Will ship sometime in Q4'2019.
+	// TODO[pulumi-service#2224]: UI support for moving stacks between organizations
```

Comment:
This links to a private repository--are we OK with having links and comments like this in the code?

Reply:
I think we could reasonably decide going either way on this. So I could take it or leave it. I erred on the side of adding it just to provide a little more transparency. e.g. "This isn't supported yet" vs. "Sometime in Q4" vs. "Here is the issue (that you probably cannot see), but no seriously, we're working on it".

Reply:
I think we should definitely remove the date. If it were me, I'd remove the issue link as well, but I don't feel that strongly.

Reply:
Sounds good to me, will remove the date and issue link.